### PR TITLE
Update README.en.md: translate some Chinese content

### PR DESCRIPTION
**Pull request by QilanLin**

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -158,9 +158,9 @@ Bad:
 
 ## Nouns
 
-### 專有名詞使用正確的大小寫
+### Use proper upper/lower cases for specialized terms
 
-大小寫相關用法原屬於英文書寫範疇，不屬於本 wiki 討論內容，在這裡只對部分易錯用法進行簡述。
+The relative usage of upper/lower cases belongs to the realm of English orthography, and it is out of the scope of this wiki. Only some common bad usages are hereby briefly discussed.
```
Comment (liyishuai) [inline README.en.md:163]:
```suggestion
The use of upper/lower cases is a matter of English orthography, which is beyond the scope of this wiki. Here we exemplify some typical scenarios.
```

File: README.en.md [hunk 2]
```diff
@@ -158,9 +158,9 @@ Bad:
 
 ## Nouns
 
-### 專有名詞使用正確的大小寫
+### Use proper upper/lower cases for specialized terms
```
Comment (liyishuai) [inline README.en.md:161]:
```suggestion
### Capitalize properly
```

**Review by liyishuai — COMMENTED** [review]
Verbalize for clarity and simplicity.

**Review by sparanoid — APPROVED** [review]

**Comment by QilanLin** [comment]
> Verbalize for clarity and simplicity.

翻译应做到「信、达、雅」。你的翻译没有使用学术语言且没有保留原句所传递的所有含义。本人为伦敦留学生，所使用的英文语言应具有一定权威性。请你学会尊重他人的劳动成果，若翻译得比我好那我心服口服，真的没想到你会把我翻译好的内容再重新用口语化的词汇 paraphrase 一遍。。。

**Comment by liyishuai** [comment]
> > Verbalize for clarity and simplicity.
> 
> 翻译应做到「信、达、雅」。你的翻译没有使用学术语言且没有保留原句所传递的所有含义。本人为伦敦留学生，所使用的英文语言应具有一定权威性。请你学会尊重他人的劳动成果，若翻译得比我好那我心服口服，真的没想到你会把我翻译好的内容再重新用口语化的词汇 paraphrase 一遍。。。

I happened to complete my Ph.D. in United States, where English is valued by clarity and simplicity.
If you believe your British English to be more authentic, please instruct me on improving my dissertation: https://repository.upenn.edu/edissertations/5376

**Comment by liyishuai** [comment]
> 请你学会尊重他人的劳动成果，若翻译得比我好那我心服口服，真的没想到你会把我翻译好的内容再重新用口语化的词汇 paraphrase 一遍。。。

Frankly speaking, your English sounds like machine translation. I failed to see where your confidence comes from.

I write English for my readers to understand what was going on in my mind, not for showing off how many words I have memorized before the IELTS exam.

This repository is to instruct how to write and typeset in a reader-friendly way. If you have a different understanding of what "academic" writing should look like, please send me a copy of your academic publication so that I can learn and comprehend.

**Comment by QilanLin** [comment]
Thank you for putting the accuracy of my translation on par with that of machine translation. I could not figure out why you supposed me to have shown off. I've just used the equivalent words to translate the original content. This translation is not in the cause of making the readers understand "what was going on in my mind" as you said but the authentic meaning of the original sentence in Chinese. Furthermore, if you really want to contribute something to the repository, you might start contributing from the beginning rather than being lazy to do a disservice to what others have done. And I am quite surprised that this could be related to "whether British is more authentic than American English". This is the same as you confused accuracy with what "sounds like machine translation".

**Comment by liyishuai** [comment]
I'm short of intelligence for understanding what you mean by "disservice to what others have done".
By suggesting changes, all I did was to present `@sparanoid`---the repository owner---yet another option that they (or a more proper pronoun) can choose to either include in or exclude from the merge commit. It was the repository owner's choice to prefer my translation to yours, because you failed to convince them that yours was better.

I've also failed to understand why my contribution to this repository should be classified as "being lazy".
#183 is your first ever contribution to this repository, while I've already participated in at least issues/PRs/discussions labelled {100,114,139,140,141,142,144,146,166} (not citing them with GitHub Markdown format to avoid context pollution). Not to mention my being proudly [sponsored](https://github.com/sponsors/liyishuai) by the repository owner for my continuous contributions.

Summary: Go improve your English, and go learn how GitHub PR works.

**Comment by QilanLin** [comment]
The reason I do this PR is purely out of sheer boredom and seeing there is some remaining Chinese sentences in the English version by accident. Whether you have a PhD and whether British is more authentic is irrelavant to the issue.

Summary: Go stick to the issues, and judge the case as it stands

**Comment by QilanLin** [comment]
>Not to mention my proudly being [sponsored](https://github.com/sponsors/liyishuai) by the repository owner for my continuous contributions.

> Frankly speaking, your English sounds like machine translation. I failed to see where your confidence comes from.

Not to mention, were it not for ME who found these remaining Chinese sentences in the English version and contributed purely out of sheer boredom, you would not have the chance to paraphrase my words in oral language and show off your PhD. Indeed, I do not have a PhD now when I am 18, but I suppose you did not have a PhD when you are 18 too. I have a work accepted by an international journal recently; you might check it after publication. However, I failed to see where your confidence that you thought my English is worse than you came from.

我还是那句话: 请你尊重别人的劳动成果。

IMG_6400

**Comment by liyishuai** [comment]
Yes it is my fault for proposing an alternative translation to yours, making you so angry and so eager to prove yourself to be better. I hereby declare you as the best scientist I’ve ever heard of. Do you want me to mail you a signed letter of affirmation for it?

**Comment by QilanLin** [comment]
> not for showing off how many words I have memorized before the IELTS exam.

Thank you for pointing out my mistake. Only when I write in acaemic language do I proof-read my words. Also, please kindly allow me to propose that in this quoted sentence:
- The determinter should be “an” as opposed to "the". “The” is used to refer to something identified.
- Alternatively, you may use "had" to make the pluperfect grammar correct.

I don't want to check the rest of all your words.

> Do forget about you being "an international student in London" when judging others.

What I said is "本人为伦敦留学生，所使用的英文语言应具有一定权威性"; not "本人为伦敦留学生，所使用的英文语言具有一定权威性" or “本人为伦敦留学生，所使用的英文语言具有权威性”. And I **had** already seen your PhD in your homepage before I posted. Therefore, the statement of me "judging others" is not tenable. And I used subjunctive many times during the conversation as British do as well. However, I mainly use American orthography.

> Meanwhile, don't regard your CNKI-indexed paper as an "academic contribution".

？？？

**Comment by liyishuai** [comment]
You may continue to point out my errors in either syntax, semantics, or rhetorics.
Just bear in mind that "what I typed in this discussion being bad" does not prove "your English or research to be good".

Regarding your EMFRM paper, I state it here: Your topic is useless, and your methodology (as the paper title suggests) is boring.

**Comment by liyishuai** [comment]
> What I said is "本人为伦敦留学生，所使用的英文语言应具有一定权威性"; not "本人为伦敦留学生，所使用的英文语言具有一定权威性" or “本人为伦敦留学生，所使用的英文语言具有权威性”.

This shows your Chinese not being much better than your English:
"应" is semantically overloaded in modern Chinese. While you meant "I **should (make it a rule that I)** use authentic English", your sentence conveyed to me as a reader "the English I used **should be (as you can see)** authentic". That's why I viewed you as more arrogant than you were.

**Comment by QilanLin** [comment]
> You may continue to point out my errors in either syntax, semantics, or rhetorics. Just bear in mind that "what I typed in this discussion being bad" does not prove "your English or research to be good".
> 
> Regarding your EMFRM paper, I state it here: Your topic is useless, and your methodology (as the paper title suggests) is boring.

6

**Comment by QilanLin** [comment]
> > What I said is "本人为伦敦留学生，所使用的英文语言应具有一定权威性"; not "本人为伦敦留学生，所使用的英文语言具有一定权威性" or “本人为伦敦留学生，所使用的英文语言具有权威性”.
> 
> This shows your Chinese not being much better than your English: "应" is semantically overloaded in modern Chinese. While you meant "**I** should (make it a rule that I) use authentic English", your sentence conveyed to me as a reader "**the English I used** should (as you can see) be authentic". That's why I viewed you as more arrogant than you actually were.

"应" is not semantically overloaded in modern Chinese in many cases.

**Comment by liyishuai** [comment]
Keep defending yourself. I've already delivered my sincere respect to you in https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183#issuecomment-1372675041. Now I hope you to become the advisor of my second PhD.

**Comment by QilanLin** [comment]
TLDR

**Comment by TieWay59** [comment]
> > Verbalize for clarity and simplicity.
> 
> 翻译应做到「信、达、雅」。你的翻译没有使用学术语言且没有保留原句所传递的所有含义。本人为伦敦留学生，所使用的英文语言应具有一定权威性。请你学会尊重他人的劳动成果，若翻译得比我好那我心服口服，真的没想到你会把我翻译好的内容再重新用口语化的词汇 paraphrase 一遍。。。

@QilanLin 哈喽，我是中文母语者，但不是翻译领域的专家。我可能对开源贡献有一点见解，所以来掺和一下。我看了你们的对话觉得非常有意思，经过我的阅读理解，我认为争议根源不是 https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183#pullrequestreview-1230581916 而是你在 https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183#issuecomment-1369529589 中因为误解给自己挖了一些坑。我为你罗列以下几处坑点，注意我不是跟你辩论，我是看出这里有很大误解：

-  `没有使用学术语言`：本项目的中文原文也没有使用严谨的学术语言，你可以从 liyishuai 的反馈中可以了解到这是不必要的。并且作为英语水平明显低于你们的我看来， liyishuai 的修改建议确实是更加直白。翻译的好还是不好，说实话你们从领域来说都不是翻译专业吧。你从学术的角度来考虑当然是觉得学术语言好，但是从软件产品的角度来看，会倾向于选择让受众更好理解的翻译。
- `尊重他人的劳动成果`：你认为 liyishuai 没有尊重你成果么？在我看来他的表述  `Verbalize for clarity and simplicity.`  相对于你的并没有夹带情绪，并且这个 github 功能只提供了建议而已，并不是真的把你修改了。 你可以认为他建议的对象不是针对你的，而是对仓库的维护者 sparanoid 说的，你看到 [774e8aa](https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183/commits/774e8aaf5f0c3cf97da474b0fb48284e1eb087fd) [42f8a63](https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183/commits/42f8a6393d46d0afa1b40f91287268bb76428812) 都是维护者做的操作，不是 liyishuai 的操作，只是 sparanoid 比较忙没有留下评论跟你交流而已，但是也可以理解。你对 liyishuai 有这么大的不满我觉得非常不必要，在这个整个合作过程中，liyishuai 是在帮助你让这个 PR 合入了，并且是作为你的合作者的身份，并不是你对手。你去查看主分支的 commit 历史，你的记录也存在历史里面。
- `本人为伦敦留学生，所使用的英文语言应具有一定权威性`：这句话怪在，这句话不是在围绕翻译内容来评估好坏，而是搬出自己的个人经历作为背书。你这样去说就很危险，说实话你跟 liyishuai 拼履历真的没意义，可能你当时没想那么多。如果说谁先开始偏离问题本身，你这句话就是起点。去掉这句话也不影响你的诉求，但是你加上这句话就很让人迷惑。这是我觉得整个帖子中最精彩的地方。所以也不奇怪为什么 liyishuai 在后面针对你的背景阴阳你，你真的太不谨慎了。

还有一个后文关于你 PR 的动机：

- `The reason why I do this PR is purely out of sheer boredom` 你说出这个动机是非常不好的，可以看出 liyishuai 在这个项目中投入了很多心思，并且给你建议也是在帮助你，然后你说出你贡献的动机是这样，还因为误会而说了很多莫名其妙的话。从感性的角度来思考，这就是在引起贡献者的不满。

TLDR： liyishuai 的第一个建议并没有冒犯你，是仓库维护者选择了接受他的建议，从结果来说是帮助了你。你对翻译的选择有诉求其实应该拉出维护者来理论，而不是和 liyishuai。这里就体现你对 Github PR 的不了解。还有无论做什么事情，尽量围绕事情本身，千万不要把自己的背景拿出来作为依据，那只会对自己不利。

后续你们的对话就是基于已有的误解相互伤害了，都是不应该的，所以我也不评价。

我说这些的动机是，喜欢吃🍉并且本身也从事一些开源社区的观察与案例研究。请问 @QilanLin @liyishuai 你们是否介意我把这个 PR 作为案例分享给我的同事或者学生？我会避开后续的关于个人信息的辩论。只围绕 GitHub 协作的思考。

**Comment by liyishuai** [comment]
> 我说这些的动机是，喜欢吃🍉并且本身也从事一些开源社区的观察与案例研究。请问 @QilanLin @liyishuai 你们是否介意我把这个 PR 作为案例分享给我的同事或者学生？我会避开后续的关于个人信息的辩论。只围绕 GitHub 协作的思考。

Despite the public nature of this repository, I hereby approve quotations to whatever I wrote here, anonymized or not.

**Comment by QilanLin** [comment]
你当然可以感性，我当然也可以。他可以追求简洁的语言风格，我也可以追求“学术语言”。
他可以觉得我的语言像机翻，我也可以觉得他的语言太oral。我所使用的语言确实更正式，至于所谓“搬资历”，完全是断章取义，请把整篇对话看完，不止一处可以驳斥你的猜测。

1. 我写的是“应具有一定”，这是一个很明显的hedging。
2. 我在回复他之前就看了他的主页

你若觉得我对他不尊重我劳动成果的 rebuttal 是“太不谨慎了”，他一开始先不尊重我的劳动成果难道不也是“太不谨慎了”吗？而且你这句“你真的太不谨慎”的说教，按你的观感，应该也是“太不谨慎了”罢。
你说你做这些的动机是吃瓜，你觉得你的动机没问题，没有不尊重任何人。我做这个PR的动机也是 purely out of sheer boredom，我也觉得我的动机没问题，没有不尊重任何人。

只要不违背公德不违法，没有人说话绝对必须“谨慎”、事先自我审查、看资历察言观色（当然从主页来看，我的计算机资历比你们两个低），请尊重每个人的言论自由，即使观点不同。我从一开始就这样做了，希望您也一样。@tieway59 
我授权此次非商业分享，匿名与否随意，前提是必须展示我写的这段话。我可能回复晚了，如果已经分享完了，也请再向你的分享对象展示这段话。

**Comment by liyishuai** [comment]
> 你若觉得我对他不尊重我劳动成果的 rebuttal 是“太不谨慎了”，他一开始先不尊重我的劳动成果难道不也是“太不谨慎了”吗？而且你这句“你真的太不谨慎”的说教，按你的观感，应该也是“太不谨慎了”罢。

如 https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183#issuecomment-1574658786 所述：
> 这个 github 功能只提供了建议而已，并不是真的把你修改了。 你可以认为他建议的对象不是针对你的，而是对仓库的维护者 sparanoid 说的

我在2022年提出的所谓“建议”，你应视为“修订案之修订案”，并把sparanoid的行为理解为两部分：
1. 接受了你的修订案，并且
2. 接受了我在其基础上的进一步修订

至于2的原因，则是维护者认为：我的进一步修订相较“经你修订后的版本”更优。故曰：
> 你对翻译的选择有诉求其实应该拉出维护者来理论，而不是和 liyishuai。

也是因此我在 https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183#issuecomment-1372113030 里讲：
> Summary: Go improve your English, and go learn how GitHub PR works.

---
> 我写的是“应具有一定”，这是一个很明显的hedging

在你看来`很明显`不蕴含其对读者“同样明显”。
如 https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183#issuecomment-1373166467 所述，在看到你用英语解释`应`的语义之前，我与TieWay59一样，将你的发言视为“晒履历”——此一句话造成两个人的误解，足以反映你在写那段文字时，未充分考虑汉语多义的内禀属性给读者带来的潜在影响，此之谓`不谨慎`。

**Comment by TieWay59** [comment]
hi @QilanLin ，我当时想法很罗嗦太像说教，很抱歉哦没照顾到你感受。

我比较关注的是以下两个回复，既然你觉得我断章取义，看看最开始是怎么样的：

>               Verbalize for clarity and simplicity.
>  
> _Originally posted by liyishuai in https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183#pullrequestreview-1230581916_

>               > Verbalize for clarity and simplicity.
> 
>                翻译应做到「信、达、雅」。你的翻译没有使用学术语言且没有保留原句所传递的所有含义。本人为伦敦留学生，所使用的英文语言应具有一定权威性。请你学会尊重他人的劳动成果，若翻译得比我好那我心服口服，真的没想到你会把我翻译好的内容再重新用口语化的词汇 paraphrase 一遍。。。
> 
> _Originally posted by QilanLin in https://github.com/sparanoid/chinese-copywriting-guidelines/issues/183#issuecomment-1369529589_

这里第一次回应的说教语气也没我差吧，“请你学会” 语气比我厉害多了哦。

我的核心看法是：liyishuai 给出 review ，被维护者 sparanoid 采纳，liyishuai 没有不尊重你的成果。很多人都可以给 PR review，而维护者有能力无视 review。

如果真要找目标，不应该是维护者 sparanoid 责任最大么。他直接就采用了 liyishuai 的 review。是 sparanoid 执行了修改，而不是其他人。所以真正不尊重你劳动成果的是不是 sparanoid，因为 sparanoid 为什么不无视 liyishuai 的 review 直接合并你的呢？

在你首次回应之前，实际上是 liyishuai 帮助了你。而你对翻译风格和质量的疑问，应该 rebuttal 的对象是 sparanoid，而非 liyishuai，因为 sparanoid 完全可以拒绝他。而 sparanoid 的评价标准是怎么样的，你和 liyishuai 都不完全清楚吧。

**Comment by liyishuai** [comment]
> 但这不代表我作为新的 contributer 不能对 liyishuai 重新用口语化的词汇 paraphrase 我反复揣摩翻译好的劳动成果这件事 raise a dispute

以我对TieWay59的评论之理解，其批判的正是你（QilanLin）dispute错了对象：
> 请你学会尊重他人的劳动成果，若翻译得比我好那我心服口服，真的没想到你会把我翻译好的内容再重新用口语化的词汇 paraphrase 一遍。。。

如果我的汉语达标，那么此处的`你`大概指的是liyishuai而非sparanoid吧？
倘若上述推测无误，则单此一句就暴露了QilanLin对GitHub机制的不了解。
在此基础上，liyishuai选择了阴阳怪气，TieWay59选择了细心讲解，QilanLin选择了继续辩护：我们都有美好的未来。

**Comment by TieWay59** [comment]
@QilanLin 我给你概括一下你在干神魔：你和不认识的同事一起作了贡献，最后主管默默采纳了同事的（实际上也有你的功劳），然后你找同事半天辩论半天为什么你的更好。

liyishuai 再有利益相关，他依旧是没权限的呀。

**Comment by QilanLin** [comment]
> 最后主管默默采纳了同事的（实际上也有你的功劳），然后你找同事半天辩论半天为什么你的更好。

不行吗

**Comment by QilanLin** [comment]
> 然后你找同事半天辩论半天为什么你的更好

不行吗? liyishuai 把我的书面语言改成口头语言, 而且就留了一句 Verbalize for clarity and simplicity, 没有说服力, 这是 freerider 的行为, 我为什么不能辩论?

> liyishuai 再有利益相关，他依旧是没权限的呀。

但是他就是有利益相关的, sparanoid 可能更信任他, 但这不代表新的 contributer 不能发言, 之前 liyishuai 也拿“我是新的 contributer”来说事, 论资排辈

**Comment by liyishuai** [comment]
> 之前 liyishuai 也拿“我是新的 contributer”来论资排辈

您的记忆模糊了，我来带您回顾一下：
> Furthermore, if you really want to contribute something to the repository, you might start contributing from the beginning rather than ***being lazy*** to do a disservice to what others have done.

我的回应只为了批判你的`lazy`一词，没有`新的 contributer 不能发言`的意思：
> I've also failed to understand why my contribution to this repository should be classified as "being lazy".
https://github.com/sparanoid/chinese-copywriting-guidelines/pull/183 is your first ever contribution to this repository, while I've already participated in at least issues/PRs/discussions labelled {100,114,139,140,141,142,144,146,166} (not citing them with GitHub Markdown format to avoid context pollution).

---
> 但是他就是有利益相关的, sparanoid 可能更信任他

恕我愚钝：我在GitHub以外的场合与sparanoid毫无交集，甚至不知该用哪一人称代词：
> all I did was to present to `@sparanoid` yet another option that ***they (or a more proper pronoun)*** can choose

赞助liyishuai是sparanoid的自发行为；在本专案的全部贡献都是liyishuai的自发行为。何谈`利益相关`？

**Comment by QilanLin** [comment]
> 而 liyishuai 在我之前已经 PR 了多次, sparanoid 可能更信任他

**Comment by liyishuai** [comment]
> > 而 liyishuai 在我之前已经 PR 了多次, sparanoid 可能更信任他
> 
> 而且他赞助了你, 难道不优先选你的? (以上仅为鄙人按经济规律常识所做之猜测)

容我解读一下您的逻辑链：
1. sparanoid赞助过liyishuai；
2. 因为(1)，所以sparanoid相比于QilanLin，更加信任liyishuai；
3. 因为(2)，所以sparanoid优先选择liyishuai的提案。

从(2)到(3)的推理，其隐含的假定是：“sparanoid会根据其对**贡献者**的信任程度，决定采用哪一**贡献**”。
这在开源社区是非常严重的指控，等于在宣布：“sparanoid是个不专业的管理者。”

如果您对我的指控是“**利用sparanoid的不专业性，凭借其对liyishuai的信任**，用liyishuai的贡献取代QilanLin的贡献”，那么我拒绝此指控——因为我从不认为sparanoid会根据**贡献者**是谁来决定采用哪一**贡献**，并主张此指控源自你的恶意揣测。

**Comment by QilanLin** [comment]
> sparanoid 可能更信任他
我说的是“可能”

**Comment by QilanLin** [comment]
你爱这样恶意揣测就这样吧, 不要把责任转移到别人身上, 我也从来没有说过这样的话或表达 sparanoid 具有所谓的“不专业性”. 正如你所说, 这只是你 “隐含的假定“
您所谓的逻辑链也是错的, 这表明您的中文阅读能力还有待提高

1. liyishuai 在我之前已经 PR 了多次
2. 因为(1)，所以sparanoid可能更加信任liyishuai  (我说的是“ “可能” 更加信任liyishuai”, 而你直接就指控 “sparanoid会根据贡献者对其的贡献程度，从而更加信任某一贡献者”, 如果你觉得你的这个指控没问题, 那么你逻辑链的2就是能推理到3的, 人都不要装得太清高)；
3. sparanoid赞助过liyishuai
4. 因为 2 3 , 所以sparanoid优先选择liyishuai的提案。

而且这不是什么“指控”, 只是按经济规律常识所做的猜测而已, 
您觉得我的翻译“sounds like machine translation”, 没有符合你自己的想法 “what was going on in my mind” 就来改我的劳动成果 难道不是恶意揣测? 你都实际这样做了还不让别人猜啊 我被无理由拒绝了也有权猜测, 如果你觉得这不正常, 那请您这辈子避免进行任何猜测, 因为都会造成对别人的不尊重 😁
如果您觉得猜测是“很严重的指控” 等于在宣布：“sparanoid是个不专业的管理者”, **那您不应该更觉得你自己用 oral language 来 paraphrase 我辛苦的劳动成果 这一实际行为 绝对等于 对我的不尊重吗?** 

您说您是被 sparanoid proudly sponser 的, 而我是存粹用爱发电给 sparanoid 的仓库做贡献的, 如果您觉得sparanoid 的仓库的 contributer 对 sparanoid 有天然的恶意, contributer 对 sparanoid 的猜测也有恶意那请 @sparanoid 赶紧把我的这个 PR 删掉吧.

**Comment by liyishuai** [comment]
> 如果您觉得sparanoid 的仓库的 contributer 对 sparanoid 有天然的恶意, contributer 对 sparanoid 的猜测也有恶意那请 `@sparanoid` 赶紧把我的这个 PR 删掉吧.

1. 您说有没有可能，我指也许，liyishuai在本专案里的角色，是个与您完全对等的contributor（注意拼写啊喂）？
2. 您说有没有可能，我指也许，身为contributor的liyishuai，没有能力也没有意愿，去指使身为owner的sparanoid做任何事情？
3. 您说有没有可能，我指也许，PR不是用来删的，而是用来保存全部变更及其相关讨论的？
---

> 而且这不是什么“指控”, 只是按经济规律常识所做的猜测而已

~您的经济规律常识，是跟A股学的吗？~

**Comment by QilanLin** [comment]
>  是个与您完全对等的contributor（注意拼写啊喂）？

这个不重要, _Just bear in mind that "what I typed in this discussion being bad" does not prove "your English or research to be good.“_,  and does not prove your English translation to be better than mine as well.
> 您说有没有可能，我指也许，身为contributor的liyishuai，没有能力也没有意愿，去指使身为owner的sparanoid做任何事情？

我针对的有且仅有 Dr. liyishuai 用 oral language 来 paraphrase 我辛苦的劳动成果 这件最先发生的事, 至于 sparanoid 选谁, 那是因为这件事情才有的.

**Comment by QilanLin** [comment]
@TieWay59

> The reason why I do this PR is purely out of sheer boredom 你说出这个动机是非常不好的，可以看出 liyishuai 在这个项目中投入了很多心思，并且给你建议也是在帮助你，然后你说出你贡献的动机是这样，还因为误会而说了很多莫名其妙的话。从感性的角度来思考，这就是在引起贡献者的不满。

我 _proudly_ 承认, 我做这个PR的动机就是 purely out of sheer boredom, 但我花时间认真阅读了许多文件才能找到隐蔽的遗漏的中文, 如果不是 purely out of sheer boredom, 我也不会花时间去翻且找到了隐蔽的遗漏的中文, 我的动机哪里不好了? 从感性的角度来思考，我也是贡献者, 我认真阅读了许多文件且细心精准翻译投入的心思绝对不是微不足道的, 您引起了我的不满。

**Comment by liyishuai** [comment]
> 我针对的有且仅有 Dr. liyishuai 用 oral language 来 paraphrase 我辛苦的劳动成果 这件最先发生的事, 至于 sparanoid 选谁, 那是因为这件事情才有的.

容我理解一下：您主张“Dr. liyishuai提出一种异于QilanLin的翻译方式是不道德的”，是这意思吗？
如果我的上述理解正确，则存在两种解释：
1. 任何人提出异于QilanLin的翻译方式都是不道德的；
2. 只有Dr. liyishuai这样做才是不道德的，别人随意。

请问您指哪种？

**Comment by QilanLin** [comment]
任何人来改都没问题, 但前提是不能乱改. 我一开始就说了:
> ### 若翻译得比我好那我心服口服
> ，真的没想到你会把我翻译好的内容再重新用口语化的词汇 paraphrase 一遍。。。

如果您有因看见我的翻译没有符合你自己的想法就来随便乱改的权利, 那么我更有表达自己的劳动成果受到不尊重的权利, 下面就来带您回顾一下您是怎么随便乱改的:

原文: 專有名詞使用正確的大小寫
- 鄙人: Use proper upper/lower cases for specialized terms
- Dr. liyishuai改完后: Capitalize properly

原文: 大小寫相關用法原屬於英文書寫範疇，不屬於本 wiki 討論內容，在這裡只對部分易錯用法進行簡述。
- 鄙人: The relative usage of upper/lower cases belongs to the realm of English orthography, and it is out of the scope of this wiki. Only some common bad usages are hereby briefly discussed.
- Dr. liyishuai改完后: Capitalization is a matter of English, which is beyond the scope of this wiki. Here we exemplify some typical scenarios.

无意冒犯, 但我也不想撒谎. 知道做出上述行为⬆️, 可能花了不到10秒钟用 oral language 来 paraphrase 我细心翻译的内容的竟然是一个 美国 PhD 反而让我想笑.

这样用 oral language 来 paraphrase 我辛苦的劳动成果 并且 只留了 5 个单词来进行主观的解释 就是一个很不礼貌的行为 也是对我劳动成果的不尊重.

**Comment by liyishuai** [comment]
> 无意冒犯, 但我也不想撒谎. 知道做出上述行为⬆️, 可能花了不到10秒钟用 oral language 来 paraphrase 我细心翻译的内容的竟然是一个 美国 PhD 反而让我想笑.

您想笑就大声笑，不会改变你的提案未被采纳之事实。
你的提案未被采纳，与我的身份是美国PhD还是英国留学生无关，纯是因为你的翻译不如我的合乎owner之喜好。

> 这样用 oral language 来 paraphrase 我辛苦的劳动成果 并且 只留了 5 个单词来进行主观的解释 就是一个很不礼貌的行为 也是对我劳动成果的不尊重.

你辛苦归辛苦，翻得烂归翻得烂。我不否认你在翻译中付出的汗水，但不会改变我认为“你用汗水换来的是一坨不会被采纳的垃圾”之想法。

以上。你想批判我的行为，或者批判我的思想，都是你的自由。至于我听到您的高谈阔论之后能否被打动，那就看您的雄辩了。

**Comment by QilanLin** [comment]
1 Use proper upperlower cases for specialized terms
orthography, and it is out of the scope of this wiki  Only some common bad usages are


您有 owner 做背书, 我有 GPT-4 做背书, _“我们都有光明的未来.”_ 
我对 owner 没有恶意, 但私以为 GPT-4 在这个情景下的判断比 liyishuai TieWay59 sparanoid QilanLin 都准确

我不否认你对这个仓库的多次贡献，但不会改变“你可能花了不到10秒钟用 oral language 来 paraphrase 我细心翻译的内容”的事实, 也不会改变我的翻译客观 (或在 AI 的主观) 上比你好的事实

如果您觉得我的截图是假的, 您可以马上复现一下这个实验结果不谢.

**Comment by liyishuai** [comment]
> 我对 owner 没有恶意, 但私以为 GPT-4 在这个情景下的判断比 liyishuai TieWay59 sparanoid QilanLin 都准确

您好，欢迎来到sparanoid主持维护的chinese-copywriting-guidelines专案。
GitHub是一个开放的代码社区。您可以自由地fork（分叉）代码仓库，并在自己的分叉中自由地修改其内容。
如果社区认为您的分叉比sparanoid更好，那么社区会自发地推崇您的分叉。
虽然我觉得您是个天才，但很遗憾，我在社区不具备将您捧上神坛的影响力。希望您能够继续努力，成为我们所有人的神。

**Comment by QilanLin** [comment]
> 至于我听到您的高谈阔论之后能否被打动，那就看您的雄辩了。

看来您已经被 GPT-4 打动了, 我接下来还有几句话想对 @TieWay59 说:

> 本项目的中文原文也没有使用严谨的学术语言，你可以从 liyishuai 的反馈中可以了解到这是不必要的。

至于我的翻译该不该叫“学术语言”这另说, 但我就是按照原文的准确意思翻译的, 要不然 liyishuai 为什么会觉得 sounds like machine translation 呢?  如果要谈翻译时的中英文行文风格差异, 对于这两句原文, 差异也不大.

> 并且作为英语水平明显低于你们的我看来， liyishuai 的修改建议确实是更加直白。翻译的好还是不好，说实话你们从领域来说都不是翻译专业吧。

鄙人不是翻译专业的但也对中英文差异有研究, 要不然也不会关注到这一个仓库, 至于谁的翻译更适合, please refer to ChatGPT above

> 但是从软件产品的角度来看，会倾向于选择让受众更好理解的翻译。

Readability 确实是一个考量因素, 但在这里 accuracy 和 clarity 比 readability 更重要, 在多种因素的综合考量下, ChatGPT 倾向于选择我的翻译. 我的版本非常准确地传达了中文原文的意思, 而且并不难以理解. 

如果您觉得我用的词汇的 readability 不高, 请您看看 README.en.md 的其他部分用的词汇 再下定论.

> 还有无论做什么事情，尽量围绕事情本身，千万不要把自己的背景拿出来作为依据，那只会对自己不利。

我在发言之前就看了 liyishuai 的主页, 他当时也在主页里写了他是美国PhD我也看到了, 我一开始的发言**围绕的就是事件本身**, 我说我的背景是“伦敦留学生”, **就是围绕 翻译 这件事情的**, 所以“应具有一定“. 所以我的背景在此就可以是一个依据, 正如⬆️ ChatGPT 给我的 endorsement 作为依据一样
我也没说我会xx编程语言, 我会xx技术 之类的话, 这个 PR 做的事情就是文科的 翻译. 

**而且, 为什么「只会对自己不利」?** 
您这样认为的话,  请先把您主页的学校删掉, 再请 liyishuai 把他主页的 PhD 删掉 因为这也算把自己的背景拿出来作为依据, 无论您在GitHub上做什么事情 别人只要把光标移到你头像上都不用点下去就能看见你的背景了😄 **这样只会对你们自己不利哦**

Regarding your 开源社区的观察与案例研究, I state it here: Your metrics are double standards, and your perspective-taking is limited.
多的我也不说了，我没义务自证给你看

**Comment by liyishuai** [comment]
> 看来您已经被 GPT-4 打动了

不，我没有。
人是万物的尺度。我对GPT的态度不会高于对yet another person的态度——择其善者而从之，其不善者而改之——除非有谁证明了“GPT永远不会胡说八道”。

至于起初请GPT给你举例子，是我懒；后续正式的修订案就是我亲自写的了。

敲这么一段只是提醒：在讨论时，“以彼之道还施彼身”未必有效，毒蛇不会被自己的毒弄死。

**Comment by QilanLin** [comment]
> 不，我没有。
人是万物的尺度。我对GPT的态度不会高于对yet another person的态度——择其善者而从之，其不善者而改之——除非有谁证明了“GPT永远不会胡说八道”。

主观的 experience 确实在个人的 “态度” 中有很大的作用, 但这不代表每个人的主观都是等效的. 有时, 客观的 metric 和共识也很重要. 至于 GPT 到底是不是 “客观” 的, 我没有说 “GPT永远不会胡说八道” 也不这么认为, 而是就事论事 _“GPT-4 在这个情景下的判断”_  _“客观 (或在 AI 的主观) 上”_

翻译和代码虽然都是编写出来的, 正如 Protagoras的这句话所表示: 不存在一个 绝对/客观 的标准来衡量事物的 真实/正确, 但翻译确是更主观的, 所以开源社区对于这两种类型的”尺度”应是不同的, 对于翻译的“尺度”要更灵活 更尊重译者的主观. @TieWay59

> 至于起初请GPT给你举例子，是我懒；后续正式的修订案就是我亲自写的了。
敲这么一段只是提醒：在讨论时，“以彼之道还施彼身”未必有效，毒蛇不会被自己的毒弄死。

2 天前 ⬆️ 的 GPT 是我自己主动“请”它的, 请翻一下对话流 您也没 让我/暗示我 这样做啊 不好意思我不知道你指的是 ? ?

**Comment by liyishuai** [comment]
> 2 天前 ⬆️ 的 GPT 是我自己主动“请”它的, 请翻一下对话流 您也没 让我/暗示我 这样做啊 不好意思我不知道你指的是 ? ?

那就是我误判了你的动机，左右横跳切换正经与不正经语气整得脑子不大灵光。鉴于现已达成粗略共识，我可以撤出该thread了。
